src/services/MemberService.java

```diff
@@ -75,13 +84,50 @@ public void update(Member obj) throws SQLException {
 
     @Override
     public Member get(Member obj) throws SQLException {
-	throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+
+        String condition = "";
+        if (obj.getId() != 0) {
+            condition = "Where id = " + obj.getId();
+        } else if (obj.getPseudo() != null) {
+            condition = "Where pseudo = " + obj.getPseudo();
+        } else if (obj.getEmail() != null) {
+            condition = "Where email = " + obj.getEmail();
+        }
+        String req = "Select * , date_format( birth_date ,'%d %m %Y') as birth from user " + condition;
```

Comment:
now that we're using java.sql.Date we shouldn't encounter problems regarding dates. So you should instead use a regular select query with no date_format function involved. Then to extract the birth_date column, just use rs.getDate("birth_date");